src/kernel/mconfig_dot.ml

```diff
@@ -284,6 +296,8 @@ let get_config { workdir; process_dir; configurator } path_abs =
         "A problem occured with merlin external configuration reader. %s If \
          the problem persists, please file an issue on Merlin's tracker."
         (match configurator with
+        | Dot_merlin_custom ->
+          Printf.sprintf "Check that `%s` is installed." Configurator.dot_merlin_custom
```

Comment:
```suggestion
          Printf.sprintf "Check that `%s` is installed."
            Configurator.dot_merlin_custom
```

Reply:
Do you have an `ocamlformat` config file or something to enforce your style ?

Reply:
We do not use ocamlformat (yet).
We don't have strict style guidelines but we do wrap lines at 80 chars.